pkg/kubelet/kubelet.go

```diff
@@ -116,6 +116,9 @@ const (
 	// Period for performing global cleanup tasks.
 	housekeepingPeriod = time.Second * 2
 
+	// Period for performing eviction monitoring.
+	evictionMonitoringPeriod = time.Second * 10
```

Comment:
Do we expose that flag?  I would like to keep it common but meant to leave a todo here for doing just that.